test/unit/src/server/handlers/t_login.js

```diff
@@ -23,31 +29,19 @@ describe('processVerifiedAssertion', () => {
     processedError = null;
   });
 
-  it('passes through verification errors', () => {
-    const error = new Error('test');
-    processVerifiedAssertion({}, res, next, error);
-    expect(processedError).toBe(error);
-  });
-
-  it('produces E_SSO_FAIL error if not authenticated', () => {
-    processVerifiedAssertion({}, res, next, null, { 'authenticated': false });
-    expect(processedError.message).toBe(constants.E_SSO_FAIL);
-  });
-
-  it('produces E_NO_SESSION error if the request has no session', () => {
-    processVerifiedAssertion({}, res, next, null, { 'authenticated': true });
-    expect(processedError.message).toBe(constants.E_NO_SESSION);
-  });
-
-  context('with OPENID_TEAMS', () => {
-    before(() => {
-      const overrides = conf.stores['test-overrides'];
-      overrides.set('OPENID_TEAMS', ['team-name']);
-    });
-
-    after(() => {
-      const overrides = conf.stores['test-overrides'];
-      overrides.clear('OPENID_TEAMS');
+  context('when OPENID_TEAMS are configured', () => {
+    let processVerifiedAssertion;
+
+    beforeEach(() => {
+      processVerifiedAssertion = proxyquire(
+        '../../../../../src/server/handlers/login',
+        {
```

Comment:
It looks cumbersome a bit, but fortunately it's just in tests.

Maybe (in undefined future) we could add some 'override' layer to the config? Just as a test affordance, so we could do something like:

```js
before(() => {
    conf.setOverrides({ UBUNTU_SSO_URL, OPENID_VERIFY_URL, OPENID_TEAMS, GITHUB_API_ENDPOINT });
});

after(() => {
    conf.clearOverrides();
});
```